docs/ONNX_Runtime_Server_Usage.md

```diff
@@ -2,7 +2,8 @@
 
 # How to Use ONNX Runtime Server for Prediction
 
-ONNX Runtime Server provides an easy way to start an inferencing server for prediction with both HTTP and GRPC endpoints. The CLI command to start the server is shown below:
+ONNX Runtime Server provides an easy way to start an inferencing server for prediction with both HTTP and GRPC endpoints. The CLI command to start the server is shown below: 
+If you built onnxruntime from source the path to onnx runtime server on linux is: ./onnxruntime/build/Linux/RelWithDebInfo/onnxruntime_server
```

Comment:
I think we better use `<Build_Config>` to replace `RelWithDebInfo`, in case you make a debug build, the path here is not right.